source/IAccessibleHandler/__init__.py

```diff
@@ -240,7 +240,10 @@
 }
 
 
-def normalizeIAccessible(pacc, childID=0):
+def normalizeIAccessible(
+		pacc: Union[IUnknown, IA.IAccessible, IA2.IAccessible2],
+		childID=0

```

Comment:
childID needs a type

```suggestion
		childID: int = 0
```